sql/civicrm_data/civicrm_extension.sqldata.php

```diff
@@ -39,6 +39,14 @@
     ],
 
     // The following should be kept in sync with the *default* list of components (`enable_components`).
```

Comment:
This comment doesn't make sense anymore.

Reply:
Conjecture: All comments eventually become false. Including this comment that all comments become false.

Reply:
I would leave a comment agreeing with you, but that too would be falsified at some point